source/_config.yml

```diff
@@ -582,6 +590,19 @@ documents:
     version: "1"
     nav: /_includes/hiera1.html
 
+  /puppetserver/6.3:
+    doc: puppetserver
+    version: "6.3"
+    nav: /puppet/6.3/_puppet_toc.html
+    external_source:
+      repo: git://github.com/puppetlabs/puppetserver.git
+      commit: origin/master
+      subdirectory: documentation
+    my_versions:
+      puppet: "6.4"
+      facter: "3.13"
```

Comment:
Again, this version gets Facter 3.13, but those docs no longer come from GitHub or have their own URL. Should this line be removed?